### PR TITLE
sfactor and housekeeping

### PR DESCRIPTION
**Pull request by VorTechnix**

### CODE REVIEW

File: worldeditadditions_commands/commands/selectors/sfactor.lua [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	description = "Make the length of one or more target axes of the current selection to be multiple(s) of <factor>.",
+	privs = { worldedit = true },
+	require_pos = 2,
+	parse = function(params_text)
+		local parts = wea.split(params_text, "%s+", false)
+		
+		--- Dissable command for current release
+		if false then
+			return false, "Feature not yet implemented."
+		end
```
Comment (sbrl) [inline worldeditadditions_commands/commands/selectors/sfactor.lua:18]: I've done the release, so it's fine if  it doesn't work properly just yet - just as long as Minetest doesn't crash on startup and any bugs are unlikely to affect (too many) other commands :-)

**Review by sbrl — COMMENTED** [review]
Great progress so far!

File: worldeditadditions_commands/commands/selectors/sfactor.lua (hunk 1 above)
Comment (VorTechnix) [reply worldeditadditions_commands/commands/selectors/sfactor.lua:18]: I'll get rid of that... I should probably take the tests out of smake too.

**Review by sbrl — APPROVED** [review]
Awesome!

**Pull request merged**